### ISSUE TITLE
Order Confirmation Email

### ISSUE THREAD

**Issue by InspireSupplements** [nozzlegear/ShopifySharp#134]
When creating an order with ShopifyOrderCreateOptions() {SendReceipt = true}

An order confirmation email is not being sent. Does anybody know why or how to get an order confirmation to send?

order = await service.CreateAsync(order, new ShopifyOrderCreateOptions() { SendReceipt = true, });

**Comment by nozzlegear** [comment]
Honestly I'm having the opposite problem; I can't get Shopify to *not* send emails when an order is created! I'll take a closer look at what's going on as soon as I get a chance.

**Comment by InspireSupplements** [comment]
Just to clarify, 

When I create an order, I get an email saying "Customer X has created an order." What I am looking to have happen is to have the customer get a confirmation email. When looking at the shopify JSON api it says:

> In addition to the properties defined earlier, you can use the following flags
> 
> send_receipt - Determines whether an order confirmation will be sent to the customer. Default: false.
> send_fulfillment_receipt - Determines whether a fulfillment confirmation will be sent to the customer. Default: false.
> inventory_behaviour - Determines which inventory updating behaviour is used. The following values are available:
> bypass (default): Do not claim inventory.
> decrement_ignoring_policy: Ignore the product's inventory policy and claim amounts no matter what.
> decrement_obeying_policy: Obey the product's inventory policy.

Let me know if you need any more information! Thank you for looking into this.

**Comment by nozzlegear** [comment]
Okay, I've found the problem. When I created the order service, I tried to be really fancy and combine the `ShopifyOrder` and the `ShopifyOrderCreateOptions` like this:

```cs
//Build the request body
Dictionary<string, object> body = new Dictionary<string, object>(options?.ToDictionary() ?? new Dictionary<string, object>())
{
    { "order", order }
};
```

Which sends the following JSON:

```json
{
    "send_receipt":true,
    "send_fulfillment_receipt":false,
    "order": { ... }
```

But the creation options are supposed to be *inside* that `order` element, not outside it. There are a couple other options that combine in the same way, so I'll hunt them down and get an update pushed out that should fix this.
